bin/ttt

```diff
@@ -7,13 +7,15 @@ require 'game_io'
 require 'wrapper_stdio'
 require 'player_human'
 require 'player_ai_basic'
+require 'player_ai_minimax'
 require 'game_rules'
 
 io = GameIo.new(WrapperStdio.new)
 
 p1 = PlayerHuman.new('X', io)
-p2 = PlayerAiBasic.new('O', GameRules.new)
+#p2 = PlayerAiBasic.new('O', GameRules.new)
```

Comment:
Remember that you can always look back in the git history for a previous implementation—no need to commit this comment.